### ISSUE TITLE
Fuel burning[BUG]

### ISSUE THREAD

**Issue by olivers1989** [Project-Mega-Pack/amp-aircraft-a330-300#68]
Hello, i have planned a flight via Simbrief.. They told me a tripfuel of 54888kg ... now i have 1hour left to fly and ive burned 30.000kg so that means ... that the difference is very very much ....

**Comment by olivers1989** [comment]
Hey, you can found the real weight on this page https://alliknowaviation.com/2019/12/14/fuel-consumption-aircraft/

its in kg.... so 5700kg per hour .... that means 12.566 lbs/per hr

you can also find it on this page https://www.airliners.net/forum/viewtopic.php?t=1355819       its the same value ...

or you can find it here https://www.quora.com/How-much-fuel-does-a-Boeing-747-or-777-consume-per-hour-while-in-flight-and-what-is-the-approximate-cost-of-that-fuel-for-the-airline         its a little bit different ... only 5.460 kg per hour ...

**Comment by aidanmacdougal** [comment]
I feel this could be related to the overpowered engine thrust as cited in issue #16 as I have noticed that during cruise it only requires around 60% N1 to stay aloft when I believe it should be around 70 or 80. So it may not necessarily be an issue with the fuel consumption itself but rather low fuel usage due to high thrust.

**Comment by aidanmacdougal** [comment]
So, I did some math and I found that this issue is actually quite egregiously bad. 

During cruise, let's generously assume that the aircraft burns 4000 lbs of fuel per hour (in reality it burns even less, more like 3700). The max allowable fuel is 235,000 lbs. This means that the aircraft could theoretically fly for **58 hours and 45 minutes.** That is beyond ridiculous and in my opinion this issue should be **high priority** and not **low priority.**

Another example. I am flying from Brisbane to Narita. According to simbrief, it requires 116,845 lbs to make the trip. Due to additional fuel requirements, the amount of fuel I actually bring is around 148,000 lbs. I will round the flight time up to 9 hours. Generously assuming that the burn rate is 4000 lbs per hour once again, I will only use 36,000 lbs of fuel. That is once again a **huge** difference between what it is and what it actually should be. If the fuel burn rate was roughly 12,500 lbs per hour as mentioned by @olivers1989 the resulting fuel consumption on this flight I mentioned would be 112.5 thousand lbs, far closer to the figure in my simbrief flight plan.

**Comment by olivers1989** [comment]
yeah so i tried a lot and the 12.500lbs per hour brought the results according to simbrief... there are still small differences but these are okay...

so please update the fuel burn to 5.700kg - 12.500lbs per hour...

regards oliver
